home/ivp-planning/user-interface/hierarchy-panel.md

```diff
@@ -0,0 +1,65 @@
+
+Via this panel it is possible to collect objects in [folders](#folders), to [rename](../machines/renaming-objects-and-folders.md) objects and folders, to toggle their [visibility](#visibility) or to lock them to prevent unwanted changes.
+
+![](../../../.gitbook/assets/hierarchy.jpg)
+
+## The elements of the panel:
+
+|                                                           |                                                                                                                                                   |
+| --------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ![](../../../.gitbook/assets/Hierarchy_search.jpg)        | The search bar makes it possible to only display such machines or objects which names contain the entered text (the search is not case sensitive) |
+| ![](../../../.gitbook/assets/Hierarchy_folder_button.jpg) | Adds a new folder to the hierarchy                                                                                                                |
+| ![](../../../.gitbook/assets/Hierarchy_color.jpg)         | Visualisation of the [color](../machines/highlighting-objects.md) set for the object(s)                                                           |
+| ![](../../../.gitbook/assets/Hierarchy_lock.jpg)          | Locks or unlocks the currently selected object(s)                                                                                                 |
+| ![](../../../.gitbook/assets/Hierarchy_visibility.jpg)    | Toggles the visibility of the currently selected object(s)                                                                                        |
+
+## Folders
+
+Folders can be used to structure projects. They can only be accessed in the hierarchy panel, however locking or hiding them will affect all objects inside - also in the [2D- ](../user-interface/the-2d-panel.md)and [3D-panel](../user-interface/the-3d-panel.md).
+
+### Creating folders:
+
+To create a folder, **press the "new folder" button** in the top of the hierarchy panel and a new folder will be added.
+
+![](../../../.gitbook/assets/Hierarchy_new_folder.jpg)
+
+{% hint style="info" %}
+The position of the new folder in will be in the topmost layer of the hierarchy. It can then be sorted into/under other folders or objects. 
+{% endhint %}
+
+### Moving objects in the hierarchy panel:
+
+To move objects or other object in the hierarchy, [select](../machines/selecting-and-moving-objects.md#selecting-objects-via-the-hierarchy-panel) their entries in the hierarchy panel and drag them on the folder/object name. A white frame will indicate where the entries will be moved to and they will be sorted into other folders or objects if applicable.
```

Comment:
Lets change this paragraph to:

Objects can be moved in the hierarchy with drag and drop. A white frame will indicate where the object will go. Drop an object onto another one, to make it a child of the target. 

Do we have a page that mentions parenting that we could link?

Reply:
no, we do not. was thinking yesterday where to put that (in another context) and took a note to check how often this is mentioned or to skip it alltogether. 
Will change "make it a child" to "group it together with"